File: src/main/resources/static/src/app/shared/product-selector/start-page.ts
import { JsonMember, JsonObject } from 'typedjson-npm/src/typed-json';
/**
 * Object to represent startPage of the product selector
 * @author ccasallas
 *
*/
@JsonObject
export class StartPage {
    @JsonMember
    titleText: string;
    @JsonMember
    startButtonText: string;
    @JsonMember
    productLink: string;
    @JsonMember
    notShureText: string;
}

File: src/main/java/co/com/fsighttech/imp/issuing/bo/StartPage.java
/**
 * 
 */
package co.com.fsighttech.imp.issuing.bo;

import lombok.Getter;
import lombok.Setter;

/**
 * Represents the initial page to start the product selector.
 * 
 * @author jtorres
 *
 */
@Getter
@Setter
public class StartPage {

	private String titleText;
	private String startButtonText;
	private String productLink;
	private String notShureText;

//	public StartPage(String titleText, String startButtonText, String productLink, String notShureText) {
//
//		this.titleText 	     = titleText;
//		this.startButtonText = startButtonText;
//		this.productLink     = productLink;
//		this.notShureText	 = notShureText;
//	}

}
